src/index.js

```diff
@@ -1,3 +1,41 @@
+<<<<<<< HEAD
```

Comment:
ohhhh man this probably caused some problems! but i’m happy to see you’re having merge conflicts :)